src/components/v5/common/ActionSidebar/partials/forms/ManagePermissionsForm/partials/PermissionsTable/PermissionsTable.tsx

```diff
@@ -28,17 +32,20 @@ const PermissionsTable: FC<PermissionsTableProps> = ({ name, role }) => {
   }
 
   return (
-    <div className="mt-7">
+    <div className={className}>
       {role !== USER_ROLE.Custom ? (
         <Table<PermissionsTableModel>
           verticalOnMobile={false}
           {...permissionsTableProps}
         />
       ) : (
         <Table<CustomPermissionTableModel>
-          className={clsx({
-            '!border-negative-400': !!fieldState.error,
-          })}
+          className={clsx(
+            'sm:[&_td>div]:py-2 sm:[&_td>div]:min-h-[2.875rem] sm:[&_td:nth-child(2)>div]:px-0 sm:[&_th:nth-child(2)]:px-0 sm:[&_tr>td]:border-none',
```

Comment:
is there really no simpler way of doing this? this seems breakable, maybe if we take this part into `useCustomPermissionsTableColumns` ?

Reply:
@bassgeta I can't use `useCustomPermissionsTableColumns` for that, because it's used to render the content that is inside each cell and I wouldn't be able to change/reset some styles from there